packages/mobile_app/lib/screens/daily/poll_widgets.dart

```diff
@@ -36,9 +36,9 @@ class PollSimpleWidget extends StatelessWidget {
       decoration: BoxDecoration(
           color: AppColors.userPollBackground,
           borderRadius: BorderRadius.circular(8.0)),
-      child: Column(
-        mainAxisAlignment: MainAxisAlignment.spaceAround,
-        crossAxisAlignment: CrossAxisAlignment.start,
+      child: ListView(
+        shrinkWrap: true,
+        physics: const NeverScrollableScrollPhysics(),
```

Comment:
What problem are we solving here?

Reply:
Out of boundaries on some screens. Found, when solving #57